roxie/ccd/ccdcontext.cpp

```diff
@@ -4053,6 +4071,16 @@ class CRoxieServerContext : public CRoxieContextBase, implements IRoxieServerCon
         {
             if (clusterWidth == -1)
             {
+#ifdef _CONTAINERIZED
+                /* NB: platform specs. are defined if agent is running in the context of
+                 * another engine, e.g. query has been submitted to Thor, but some code is
+                 * executing outside of it.
+                 * 
+                 * If not defined then assumed to be executing in roxie context,
+                 * where getNodes() defaults to 1.
+                 */
+                clusterWidth = topology->getPropInt("platform/@width", 1);
```

Comment:
Code below suggests that default should be numChannels?

Reply:
right, yes, I'll change, so consistent.